src/main/java/uk/gov/digital/ho/hocs/audit/export/infoclient/ExportViewConstants.java

```diff
@@ -2,11 +2,17 @@
 
 public interface ExportViewConstants {
 
-        String FIELD_ADAPTER_HIDDEN = "hidden";
-        String FIELD_ADAPTER_USER_EMAIL = "userEmailAdapter";
-        String FIELD_ADAPTER_USERNAME = "usernameAdapter";
-        String FIELD_ADAPTER_FIRST_AND_LAST_NAME = "userFirstAndLastName";
-        String FIELD_ADAPTER_TEAM_NAME = "teamNameAdapter";
-        String FIELD_ADAPTER_UNIT_NAME = "unitNameAdapter";
-        String FIELD_ADAPTER_TOPIC_NAME = "topicNameAdapter";
+    // identifies dates that have the format [0]YYYY-[0]MM-[0]DD
+    String MALFORMED_DATE_REGEX = "^0*\\d{4}\\-0*(?:[1-9]|1[0-2])\\-0*(?:[1-9]|[12][0-9]|3[01])$";
```

Comment:
As mentioned, not required as a find on `GROUPED_DATE_REGEX` would have the same effect.

Reply:
removed as suggested.